api/v1/version.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/mt-sre/addon-service/api/v1";
+
+message Version {
+  string version = 1;
+  string branch = 2;
+  string commit = 3;
+  string goVersion = 4;
+  string platform = 5;
+  google.protobuf.Timestamp buildTime = 6;
+}
+
+message VersionRequest {}
+
+service VersionService {
+  rpc Get(VersionRequest) returns (Version) {
```

Comment:
Can we instead try to use `google.protobuf.Empty`, instead of manually defining empty messages? 

https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Empty